pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <modules>
     <module>build-caching</module>
-    <module>j2cl-tools</module>
+    <module>j2cl-build</module>
```

Comment:
why rename/move this package?

Reply:
The master project is already called "j2cl-tools" and having the same name in a sub-module seems weird. Furthermore, the pom.xml in the j2cl-tools subdirectory calls the artifact in there already "j2cl-build". Maven modules refer to local sub-directories (AFAIK). So this "rename" is more a clean-up.

Reply:
Good point - I'll reconsider naming a bit, this got thrashed a bit during that refactor. The goal is that this entire repo is no longer specific to maven (though it is built with maven), but also provides libraries that let other projects do their own j2cl building/watching (such as IDE tooling, a "java/j2cl fiddle" site, etc).